himbaechel/uarch/gatemate/pack_clocking.cc

```diff
@@ -376,42 +413,33 @@ void GateMatePacker::pack_pll()
                 auto pad_info = uarch->bel_to_pad[clk->driver.cell->bel];
                 if (pad_info->flags == 0)
                     log_error("CLK_REF must be driven with CLK dedicated pin.\n");
-                clkin[die]->params[ctx->idf("REF%d", pll_index[die])] = Property(pad_info->flags - 1, 3);
-                clkin[die]->params[ctx->idf("REF%d_INV", pll_index[die])] = Property(Property::State::S0);
-                ci.movePortTo(id_CLK_REF, clkin[die], ctx->idf("CLK%d", pad_info->flags - 1));
             } else {
                 // SER_CLK
-                clkin[die]->params[ctx->idf("REF%d", pll_index[die])] = Property(0b100, 3);
-                clkin[die]->params[ctx->idf("REF%d_INV", pll_index[die])] = Property(Property::State::S0);
-                ci.movePortTo(id_CLK_REF, clkin[die], id_SER_CLK);
+                if (clk != net_SER_CLK)
+                    log_error("CLK_REF connected to uknown pin.\n");
```

Comment:
this error message should include some more context - like the name of the clock causing the problem